test/test.js

```diff
@@ -318,7 +319,31 @@ describe('sourcemap-concat', function() {
     });
   });
 
-  it('can ignore empty content', function() {
+  it('can build empty files with allowNone disabled', function() {
+    var node = concat(emptyFixture, {
+      outputFile: '/empty.js',
+      inputFiles: ['*.js']
+    });
+    builder = new broccoli.Builder(node);
+    return builder.build().then(function(result) {
+      expectFile('empty.js').in(result);
```

Comment:
Can we check its content also? To confirm that the resulting file is actually `""`?